client/backend/src/lib.rs

```diff
@@ -120,9 +134,18 @@ impl Vertex {
         }
     }
 
-    pub fn login(&mut self) -> Result<(), Error> {
+    pub fn create_user(
+        &mut self,
+        username: &str,
+        display_name: &str,
```

Comment:
I guess we should consider whether we should do global display name or rather have display names per server like discord?

Reply:
We can have both!